applications/nrf_desktop/src/util/config_channel.c

```diff
@@ -222,6 +222,7 @@ int config_channel_report_get(struct config_channel_state *cfg_chan,
 	}
 
 	if (cfg_chan->frame.status != CONFIG_STATUS_PENDING) {
+		atomic_set(&cfg_chan->status, cfg_chan->frame.status);
```

Comment:
this is needed - we must update status as otherwise we won't be able to start any new transaction